src/edu/buffalo/cse/jive/finiteStateMachine/parser/expression/temporal/EExpression.java

```diff
@@ -47,7 +132,7 @@ private Boolean evaluate(State prev, State curr, Set<State> visited, Map<State,
 		if (!currentResult && visited.add(curr)) {
 			boolean childResult = currentResult;
 			for (State next : states.get(curr))
-				if ((childResult = evaluate(curr, next, visited, states)))
+				if ((childResult = evaluate2(curr, next, visited, states)))
 					break;
 			currentResult = childResult;
 		}

```

Comment:
This is the older implementation of EExpresion. It is fired if we do not have E expression at the top level of given properties.